src/index.js

```diff
@@ -1,18 +1,25 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './globalScss/index.scss';
+
+import { Provider } from 'react-redux';
+import { store } from './store/store';
+
 import AppRouter from './routes/AppRouter';
 import { BrowserRouter as Router } from 'react-router-dom';
-ReactDOM.render(
-    <Router>
-        <AppRouter />
-    </Router>,
-    document.getElementById('root')
+
+import './globalScss/index.scss';
+
+const ROUTER = (
+	<Provider store={store}>
+		<Router>
+			<AppRouter />
+		</Router>
+	</Provider>
 );
 
-/**
- * For now, I thought it would be easier just to create an individual file for the routing system called AppRouter, instead of dismantling and reorganizing everything in App for the time being.
- */
+store.firebaseAuthIsReady.then(() => {
+	ReactDOM.render(ROUTER, document.getElementById('root'));
+});
```

Comment:
we want to make sure when visiting the app what the status of the auth, is user logged in/out.
at the rootReducer in the store there's an implementation of react-redux-firebase reducer that in any visit of the app should maintain an user status